graspologic/embed/omni.py

```diff
@@ -2,15 +2,55 @@
 # Licensed under the MIT License.
 
 import warnings
-from typing import Optional
+from typing import List, Optional
 
 import numpy as np
-from scipy.sparse import isspmatrix_csr
+from beartype import beartype
+from scipy.sparse import coo_matrix, csr_matrix, hstack, isspmatrix_csr, vstack
 
-from ..utils import import_graph, is_fully_connected
+from ..utils import average_matrices, import_graph, is_fully_connected
 from .base import BaseEmbedMulti
 
 
+@beartype
+def _get_omnibus_matrix_sparse(matrices: List[csr_matrix]) -> coo_matrix:
```

Comment:
```suggestion
def _get_omnibus_matrix_sparse(matrices: List[csr_matrix]) -> csr_matrix:
```

Reply:
i think the expected output is csr_matrix

Reply:
It is a coo, I couldn't find the expected output in scipy docs but I checked it with a debugger to verify.